.php-cs-fixer.php-lowest.php

```diff
@@ -24,12 +24,13 @@
 $config->getFinder()->notPath([
     // @TODO 4.0 change interface to be fully typehinted and remove the exceptions from this list
     'src/DocBlock/Annotation.php',
+    'src/Doctrine/Annotation/Tokens.php',
     'src/Tokenizer/Tokens.php',
 ]);
 
 $config->setRules([
-    'phpdoc_to_param_type' => true, // EXPERIMENTAL rule, helping to ensure usage of 7.0+ typing
-    'phpdoc_to_return_type' => true, // EXPERIMENTAL rule, helping to ensure usage of 7.0+ typing
+    'phpdoc_to_param_type' => true,
+    'phpdoc_to_return_type' => true,
```

Comment:
why were these comments  removed, these seems valid to me?

Reply:
I don't see the point of having them there, IMO that's information that should be in the rule description, not in the comments placed where the rule is used.